src/components/TalentPage/TalentPage.module.scss

```diff
@@ -0,0 +1,140 @@
+.btns {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    gap: 32px;
+    padding: 20px 20px;
+    line-height: 1.6;
+}
+.btn {
+    padding: 5px 20px;
+}
+.container {
+    display: flex;
+    width: 100%;
+    height: 100%;
+    padding: 20px 20px;
+}
+.name {
+    list-style: none;
+    margin-top: 50px;
+    font-weight: 800;
+}
+.skills {
+    padding: 10px;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 10px 0.5em;
+}
+.skill {
+    font-size: 17px;
+    background-color: rgb(21, 21, 21);
+    border-radius: 30px;
+
+    align-self: center;
+}
+.link {
+    display: inline-block;
+    flex-direction: row;
+    margin-top: 20px;
+    margin-right: 10px;
+}
+.talent_data {
+    flex: 30%;
+    display: table;
+    align-items: center;
+    text-align: center;
+    font-size: 20px;
+    background-image: linear-gradient(130deg, #8b56c0 50%, #a582f7 80.52%);
+
+    color: #fff;
+    padding: 30px 30px;
+    border-radius: 20px;
+}
+
+.ava {
+    border-radius: 50%;
+    border: 2px solid #fff;
+    height: 250px;
+    width: 250px;
+}
+.info {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+}
+
+.info_talent {
+    margin-top: 30px;
+    font-size: 30px;
+    text-align: center;
+}
+
+.socials {
+    width: 35px;
+    transition: all 0.1s ease-in;
+    &:hover {
+        transform: scale(1.1);
+    }
+}
+
+.about {
+    margin-left: 20px;
+    flex: 50%;
+    display: table;
+    padding: 30px 30px;
+    font-size: 20px;
+    background-color: rgb(21, 21, 21);
+    border-radius: 20px;
+}
+
+.ab_title {
+    text-transform: uppercase;
+    letter-spacing: 3px;
+    font-size: 50px;
+    font-weight: 500;
+}
+.bio,
+.additional_info,
+.conacts {
+    padding: 10px;
+}
+.about li {
+    list-style: none;
+}
+
+@media screen and (max-width: 1068px) {
```

Comment:
It is better to describe breakpoints for classes immediately below the first use of that class. Because, if the code is very large, it will be hard to read and maintain it. Divide these rules and move them to the right place